requirements.txt

```diff
@@ -3,4 +3,3 @@ oauthlib
 requests-oauthlib>=1.0.0
 Flask>=0.7
```

Comment:
Should we maybe up the Flask requirement to 1.0? It was released in April 2018. It's made it into most distributions by now.

Reply:
the change done in this ticket requires only Flask>=0.9